### DISCUSSION TITLE
CPU usage spiking on Mac M1

### DISCUSSION THREAD

**Discussion by lolbat** [Eugeny/tabby#4958] [General]
Has anyone had issues with Tabby using abnormal amounts of the CPU when in the background on an M1 Mac?

**Comment by Eugeny** [comment]
Could you please capture a performance trace while the CPU is spiking? (see developers.google.com/web/tools/chrome-devtools/evaluate-performance/reference#save)

**Reply by lolbat** [reply]
Will do
